docs/postgres/restore.md

```diff
@@ -0,0 +1,86 @@
+# Postgres - Restore Database
+
+## Database goes offline (Fire drill)
+
+Ensure snapshot exists for restore (aws console, rds -> snapshots).
+
+### Terraform Process
+
+#### Prepare for deletion
+- Source to specific env (ex: `source scripts/env/set-env.zsh exp4`)
+- Update allColors.tf
+  - Toggle delete_protection from true to false.
+- Update rds.tf
+  - Toggle prevent_destroy from true to false
+- Run deploy allColors (ex: `npm run deploy:allColors exp4`)
+
+#### Delete global cluster
+- Source to specific env (ex: `source scripts/env/set-env.zsh exp4`)
+- Update allColors.tf
+  - Comment out module "rds"
+- Update allColors/output.tf
+  - Comment out:
+  ```
+  output "rds_host_name" {
+    value = module.rds.address
+  }
+
+  output "rds_host_name_west" {
+    value = module.rds.address_west
+  }
```

Comment:
Very minor thought: The names for our east/west clusters seem to be operating on two different axes: one axis is the "main" host vs. the "read" host, the other is the "east" host vs. the "west" host. I wonder if we can choose a nomenclature in which "rds_host_name_west" is not being used as a proxy for "read," or else "rds_host_name" is not a proxy for east. (Hopefully this comment makes sense. Again, very minor.)